### PR TITLE
Fix readme.md and get-started document

### PR DESCRIPTION
**Pull request by kevindiu**
### Environment:

- Golang Version: 1.14
- Docker Version: 19.03.5
- Kubernetes Version: 1.17.3
- NGT Version: 1.9.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensure all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/user/get-started.md [hunk 1]
```diff
@@ -57,21 +24,24 @@ It uses the fastest ANN Algorithm [NGT](https://github.com/yahoojapan/NGT) to se
 
 Helm and hdf5 is required for this tutorial. If helm or hdf5 is not installed, please install [helm](https://helm.sh/docs/intro/install)) and [hdf5](https://www.hdfgroup.org/)).
```
Comment (vankichi) [inline docs/user/get-started.md]:
```suggestion
Helm and hdf5 is required for this tutorial. If helm or hdf5 is not installed, please install [helm](https://helm.sh/docs/intro/install) and [hdf5](https://www.hdfgroup.org/).
```

File: docs/user/get-started.md [hunk 2]
```diff
@@ -149,9 +119,9 @@ If you want to learn about Scylla, please refer to [the official website](https:
     ```
     </details>
 
-### Run
+### Example
```
Comment (vankichi) [inline docs/user/get-started.md]: - `Run using example code` is better.

File: docs/user/get-started.md [hunk 3]
```diff
@@ -167,169 +137,21 @@ This chapter shows how to run Vald with fashion-mnist dataset.
...
-            b, _ := json.MarshalIndent(res.GetResults(), "", " ")
-            glg.Infof("%d - Results : %s\n\n", i+1, string(b))
-            time.Sleep(1 * time.Second)
-        }
-        ```
-        </details>
+3. Perform a search action in Vald
+
+    In this example, the fashion-mnist dataset will insert into the vald cluster and perform a search using [vald-client-go](https://github.com/vdaas/vald-client-go).
+    Full example code is [`here`](../../example/client/main.go) and the explaination is [here](https://github.com/vdaas/vald-client-go/docs/users/code-explanation.md).
```
Comment (vankichi) [inline docs/user/get-started.md]: - I'm not sure that this change is consistent with what we discussed 😕. where is `code-explanation.md`?

Comment (kevindiu) [reply docs/user/get-started.md]: It moved to go client library repo. I created a PR on that repo. 
Also I suggest to move the example program to that repo.

**Comment by kpango** [comment]
Please fix again, we should discuss a lot about client libs and get-started.

**Pull request closed without merging**